test/metric_value_benchmark/metrics_value_benchmark_test.go

```diff
@@ -157,5 +158,10 @@ func shouldRunEC2Test(env *environment.MetaData, t *test_runner.TestRunner) bool
 	}
 	_, shouldRun := env.EC2PluginTests[strings.ToLower(t.TestRunner.GetTestName())]
 	_, shouldExclude := env.ExcludedTests[strings.ToLower(t.TestRunner.GetTestName())]
-	return shouldRun || !shouldExclude
+	if shouldRun {
```

Comment:
Fix issue with where include was adding all tests instead of only the tests in include